src/component/archive/ArchiveTable/ArchiveTable.tsx

```diff
@@ -0,0 +1,360 @@
+                    <ReviveArchivedFeatureCell
+                        project={original.project}
+                        onRevive={() => onRevive(original.name)}
+                    />
+                ),
+            },
+            // Always hidden -- for search
+            {
+                accessor: 'description',
+            },
+        ],
+        //eslint-disable-next-line
+        [projectId]
+    );
+
+    const {
+        data: searchedData,
+        getSearchText,
+        getSearchContext,
+    } = useSearch(columns, searchValue, archivedFeatures);
+
+    const data = useMemo(
+        () => (loading ? featuresPlaceholder : searchedData),
+        [searchedData, loading]
+    );
+
+    const [initialState] = useState(() => ({
+        sortBy: [
+            {
+                id: searchParams.get('sort') || storedParams.id,
+                desc: searchParams.has('order')
+                    ? searchParams.get('order') === 'desc'
+                    : storedParams.desc,
+            },
+        ],
+        hiddenColumns: ['description'],
+    }));
+
+    const {
+        headerGroups,
+        rows,
+        state: { sortBy },
+        getTableBodyProps,
+        getTableProps,
+        prepareRow,
+        setHiddenColumns,
+    } = useTable(
+        {
+            columns: columns as any[], // TODO: fix after `react-table` v8 update
+            data,
+            initialState,
+            sortTypes,
+            disableSortRemove: true,
+            autoResetSortBy: false,
+        },
+        useFlexLayout,
+        useSortBy
+    );
+
+    useEffect(() => {
+        const hiddenColumns = ['description'];
+        if (isMediumScreen) {
+            hiddenColumns.push('lastSeenAt', 'status');
+        }
+        if (isSmallScreen) {
+            hiddenColumns.push('type', 'createdAt');
+        }
+        setHiddenColumns(hiddenColumns);
+    }, [setHiddenColumns, isSmallScreen, isMediumScreen]);
+
+    useEffect(() => {
+        if (loading) {
+            return;
+        }
+        const tableState: Record<string, string> = {};
+        tableState.sort = sortBy[0].id;
+        if (sortBy[0].desc) {
+            tableState.order = 'desc';
+        }
+        if (searchValue) {
+            tableState.search = searchValue;
+        }
+
+        setSearchParams(tableState, {
+            replace: true,
+        });
+        setStoredParams({ id: sortBy[0].id, desc: sortBy[0].desc || false });
+    }, [loading, sortBy, searchValue]); // eslint-disable-line react-hooks/exhaustive-deps
+
+    const [firstRenderedIndex, lastRenderedIndex] =
+        useVirtualizedRange(rowHeight);
+
+    return (
+        <PageContent
+            isLoading={loading}
+            header={
+                <PageHeader
+                    title={`${title} (${
+                        rows.length < data.length
+                            ? `${rows.length} of ${data.length}`
+                            : data.length
```

Comment:
@nunogois it was more that we need to see how the search and title will work together. if we keep this short version it will work. I will create a task to align this across the entire app